src/main/resources/static/_shared.yml

```diff
@@ -782,6 +786,22 @@ components:
             - dateOfBirth
             - sex
             - status
+    PersonSummary:
+      type: object
+      properties:
+        crn:
+          type: string
+        name:
```

Comment:
I think 'name' should only be a property on the FullPersonSummary, so should be moved into the FullPersonSummary definition. This should be a 'required' field